pages/reference/OS/advanced.md

```diff
@@ -109,26 +106,31 @@ dtoverlay=i2c-rtc,ds1307
 dtoverlay=lirc-rpi
 ```
 
-This parsing will only be done if the value is a valid string; so if it doesn't begin with a quote `"`, the value will be parsed as a single string and not split into several lines. For instance `RESIN_HOST_CONFIG_dtoverlay = i2c-rtc,ds1307` will translate to:
+This parsing will only be done if the value is a valid string, so if it doesn't begin with a quote `"`, the value will be parsed as a single string and not split into several lines. For instance `RESIN_HOST_CONFIG_dtoverlay = i2c-rtc,ds1307` will translate to:
 
 ```
 dtoverlay=i2c-rtc,ds1307
 ```
 
-### Disabling rainbow splash screen
+### Disabling the rainbow splash screen
 
 To disable the Raspberry Pi rainbow splash screen, add the `disable_splash=1` entry to `config.txt`.
 
 __Note:__ This setting disables the Raspberry Pi rainbow splash screen but does not disable the {{ $names.company.lower }} logo splash screen. If you would like to replace the {{ $names.company.lower }} logo with your custom splash logo, replace `splash/resin-logo.png` located in the [boot partition][boot-partition] of the image.
 
 ### Further Reading
 
-There are more details on the options available in `config.txt` over at
-[elinux's RPi Config page][elinux].
+There are more details on the options available in `config.txt` over at [elinux's RPi Config page][elinux].
```

Comment:
The eLinux page is very out of date - you ought to be (only) linking to https://www.raspberrypi.org/documentation/configuration/config-txt/README.md

Reply:
Given this is already linked a couple times in the first paragraph I've just removed this section.